### DISCUSSION TITLE
bbox_loss missing during traing the pgd model on waymo dataset.

### DISCUSSION THREAD

**Discussion by hanssssssss** [open-mmlab/mmdetection3d#2069] [Q&A]
I used config file `pgd_r101_fpn-head_dcn_16xb3_waymoD5-fov-mono3d.py` and during training, I found that I could not see the bbox_loss on the log file. But the bbox_loss is defined in the config file as:

`loss_cls=dict(
            type='mmdet.FocalLoss',
            use_sigmoid=True,
            gamma=2.0,
            alpha=0.25,
            loss_weight=1.0),
        loss_bbox=dict(
            type='mmdet.SmoothL1Loss', beta=1.0 / 9.0, loss_weight=1.0),
        loss_dir=dict(
            type='mmdet.CrossEntropyLoss', use_sigmoid=False, loss_weight=1.0),
        loss_centerness=dict(
            type='mmdet.CrossEntropyLoss', use_sigmoid=True, loss_weight=1.0),
        use_depth_classifier=True,
        depth_branch=(256, ),
        depth_range=(0, 50),
        depth_unit=10,
        division='uniform',
        depth_bins=6,
        pred_keypoints=True,
        weight_dim=1,
        loss_depth=dict(
            type='UncertainSmoothL1Loss', alpha=1.0, beta=3.0,
            loss_weight=1.0),
        loss_bbox2d=dict(
            type='mmdet.SmoothL1Loss', beta=1.0 / 9.0, loss_weight=0.0),
        loss_consistency=dict(type='mmdet.GIoULoss', loss_weight=0.0),`

**Comment by JingweiZhang12** [comment]
Please provide more details. You can create an issue. BTW, please check whether you calculate the loss in your model forward.

**Reply by hanssssssss** [reply]
2022/12/01 09:10:27 - mmengine - INFO - Epoch(train) [10][150/10539]  lr: 8.0000e-03  eta: 1 day, 5:21:55  time: 0.6406  data_time: 0.0046  memory: 10087  grad_norm: 5.4908  loss: 2.5810  loss_cls: 0.1693  loss_offset: 0.6267  loss_size: 0.5805  loss_rotsin: 0.1304  loss_dir: 0.2044  loss_depth: 0.2011  loss_kpts: 0.0815  loss_bbox2d: 0.0000  loss_consistency: 0.0000  loss_centerness: 0.5872


Hi, the above is the information from the log file, every loss defined in the config file is shown except the loss_bbox. And I used the original config files provided by you guys(I only modify the batch size and number of gpus). Do you have any idea? Thanks!
